proto/payment/params.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package payment;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/celestiaorg/celestia-app/x/payment/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+
+  uint32 min_square_size = 1 [(gogoproto.moretags) = "yaml:\"min_square_size\""];
+  uint32 max_square_size = 2 [(gogoproto.moretags) = "yaml:\"max_square_size\""];
```

Comment:
possibly stupid question, why are we using the yaml here instead of something like jsontag? this seems to be uncommon in the sdk afaiu

Reply:
> It's standard practice from what I've seen. Also, it makes it very easy to define genesis parameters through a `config.yml` file
>
>For eg. https://github.com/osmosis-labs/osmosis/blob/main/proto/osmosis/lockup/params.proto

Reply:
Can go with jsontag if that's what we prefer

Reply:
yml makes sense too! I don't think it matters, as long as the genesis state is working